node/packages/sdk/lib/structured-log-to-event.js

```diff
@@ -118,17 +119,23 @@ const handleWarningLog = (logLineParsed) => {
 
 module.exports.attemptParseStructuredLogAndCapture = (logLine) => {
   try {
-    const logLineParsed = JSON.parse(logLine.toString());
-    if ('level' in logLineParsed) {
-      debuglog('LOG LEVEL', logLineParsed.level);
-      const logLevel = parseLogLevel(logLineParsed.level);
-      if (logLevel === 'ERROR') {
+    if (typeof logLine !== 'string') return;
```

Comment:
Not sure that we should limit this to just string types. The stdout/stderr write method can be a string or UInt8Array, seems like we should support both.

Reply:
Yes, but we know that tools we want to detect are not using those types to write structured logs

It's about speeding up the parsing process and aborting fast if we know it's not a chunk of our interest